src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -96,7 +96,18 @@ const getSkipSetupLink = (appState, linkName) => {
 };
 
 const getSignOutLink = (settings, options = {}) => {
-  if (settings?.get('signOutLink')) {
+
+  // appLoginRedirectUri to redirect to current app
+  if (settings?.get('appLoginRedirectUri')) {
```

Comment:
Turned out any option can be overridden in widget code editor, Deepa proposed the name `appLoginUri` and Bhagya agreed, see https://okta.slack.com/archives/CAZH2MWEL/p1652827056330209?thread_ts=1652281036.311819&cid=CAZH2MWEL

Reply:
Thanks for the summary, will update!